login-workflow/src/screens/CreateNewOrgScreen/CreateNewOrgScreen.tsx

```diff
@@ -0,0 +1,162 @@
+import { CreateNewOrgScreenBase } from './CreateNewOrgScreenBase';
+import { useRegistrationWorkflowContext } from '../../contexts';
+import { useErrorManager } from '../../contexts/ErrorContext/useErrorManager';
+import { useTranslation } from 'react-i18next';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import DomainIcon from '@mui/icons-material/Domain';
+
+/**
+ * Component that renders a screen for the user to enter an organization name to start the
+ * organization creation process.
+ *
+ * @param orgNameLabel label for the organization name field
+ * @param initialValue initial value for the orgName text field
+ * @param orgNameValidator function used to test the organization name input for valid formatting
+ * @param orgNameTextFieldProps props to pass to the organization name text field
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ * @param WorkflowCardActionsProps props that will be passed to the WorkflowCardActions component
+ * @param errorDisplayConfig configuration for customizing how errors are displayed
+ *
+ * @category Component
+ */
+
+export const CreateNewOrgScreen: React.FC<CreateNewOrgScreenProps> = (props) => {
+    const { t } = useTranslation();
+    const regWorkflow = useRegistrationWorkflowContext();
+    const { nextScreen, previousScreen, screenData, totalScreens, currentScreen } = regWorkflow;
+    const [organizationNameInputValue, setOrganizationNameInputValue] = useState(
+        screenData?.Other?.CreateOrganization?.organizationName || ''
+    );
+    const [isLoading, setIsLoading] = useState(false);
+    const { triggerError, errorManagerConfig } = useErrorManager();
+    const errorDisplayConfig = {
+        ...errorManagerConfig,
+        ...props.errorDisplayConfig,
+        onClose: (): void => {
+            if (props.errorDisplayConfig && props.errorDisplayConfig.onClose) props.errorDisplayConfig.onClose();
+            if (errorManagerConfig.onClose) errorManagerConfig?.onClose();
+        },
+    };
+
+    const NewOrgInstructions = (): JSX.Element => (
+        <Box sx={{ p: '16px 24px 0', display: 'flex', flexDirection: 'column' }}>
+            <Box
+                sx={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    width: '100px',
+                    height: '100px',
+                    backgroundColor: '#e0eff8',
+                    borderRadius: '50%',
+                    alignSelf: 'center',
+                    mb: 6,
+                    mt: 2,
+                }}
+            >
+                <DomainIcon color={'primary'} sx={{ fontSize: 54 }} />
```

Comment:
just wondering if this should just use icon prop so it can be changed if user passes in a different icon?